### DISCUSSION TITLE
spring-boot-2/4-9-jta

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#22] [Announcements]
# spring-boot-2/4-9-jta

Spring Boot 2.x基础教程：使用JTA实现多数据源的事务管理 在一个Spring Boot项目中，连接多个数据源还是比较常见的。之前也介绍了如何在几种常用框架的场景下配置多数据源，具体可见： Spring Boot 2.x基础教程：JdbcTemplate的多数据源配置 Spring Boot 2.x基础教程：Spring Data JPA的多数据源配置 Spring Boot 2.x基础教程：MyBatis的多数据源配置

https://www.didispace.com/spring-boot-2/4-9-jta.html

**Comment by YU-7** [comment]
勘误：“注意，这里除了家在的配置不同之外” -> 加载
